omod/src/main/resources/config.xml

```diff
@@ -265,6 +265,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>coreapps.conditionListClasses</property>
+        <defaultValue>Diagnosis</defaultValue>
+        <description>
+            List of possible classes which limits the answers for condition list
```

Comment:
Can you also change the description to clearly state that this is a list of uuids?